docs/getting-started/integrations-cdn/react-default-cdn.md

```diff
@@ -38,7 +38,238 @@ You get ready-to-use code tailored to your needs!
 
 This guide assumes you have a React project. You can create a basic React project using [Vite](https://vitejs.dev/). Refer to the [React documentation](https://react.dev/learn/start-a-new-react-project) to learn how to set up a project in the framework.
 
-## Component properties
+### Installing from npm
+
+Install the `@ckeditor/ckeditor5-react` package:
+
+```bash
+npm install @ckeditor/ckeditor5-react
+```
+
+Use the `<CKEditor>` component inside your project. The below example shows how to use the component with the open-source plugins.
+
+```js
+import React from "react";
+import { CKEditor, useCKEditorCloud } from "@ckeditor/ckeditor5-react";
+
+const CKEditorDemo = () => {
+	const cloud = useCKEditorCloud( {
+		version: "43.0.0",
+	} );
+
+	if ( cloud.status === "error" ) {
+		return <div>Error!</div>;
+	}
+
+	if ( cloud.status === "loading" ) {
+		eturn <div>Loading...</div>;
+	}
+
+	const {
+		ClassicEditor: ClassicEditorBase,
+		Bold,
+		Essentials,
+		Italic,
+		Paragraph,
+		Undo,
+	} = cloud.CKEditor;
+
+	return (
+		<CKEditor
+			editor={ClassicEditorBase}
+			data={"<p>Hello world!!!</p>"}
+			config={{
+			toolbar: {
+				items: ["undo", "redo", "|", "bold", "italic"],
+			},
+			plugins: [ Bold, Essentials, Italic, Paragraph, Undo ],
+			}}
+		/>
+	);
+};
+```
+
+To use premium plugins, set the `premium` property to `true` in the `useCKEditorCloud` configuration and provide your license key in the `CKEditor` configuration.
+
+```js
+import React from "react";
+import { CKEditor, useCKEditorCloud } from "@ckeditor/ckeditor5-react";
+
+const CKEditorDemo = () => {
+	const cloud = useCKEditorCloud({
+		version: "43.0.0",
+		premium: true,
+	});
+
+	if ( cloud.status === "error" ) {
+		return <div>Error!</div>;
+	}
+
+	if ( cloud.status === "loading" ) {
+		return <div>Loading...</div>;
+	}
+
+	const {
+		ClassicEditor: ClassicEditorBase,
+		Bold,
+		Essentials,
+		Italic,
+		Mention,
+		Paragraph,
+		Undo,
+	} = cloud.CKEditor;
+
+	const { SlashCommand } = cloud.CKEditorPremiumFeatures!;
+
+	return (
+		<CKEditor
+			editor={ ClassicEditorBase }
+			data={ "<p>Hello world!!!</p>" }
+			config={{
+				licenseKey: "<YOUR_LICENSE_KEY>",
+				toolbar: {
+					items: [ "undo", "redo", "|", "bold", "italic" ],
+				},
+				plugins: [
+					Bold,
+					Essentials,
+					Italic,
+					Mention,
+					Paragraph,
+					Undo,
+					SlashCommand,
+				],
+			}}
+		/>
+	);
+};
+```
+
+### Usage with CKBox
+
+To use `CKBox`, specify the version and theme (optionally) in the `useCKEditorCloud` configuration. Also, remember about the actual plugin configuration inside `<CKEditor/>` component.
+
+```js
+import React from "react";
+import { CKEditor, useCKEditorCloud } from "@ckeditor/ckeditor5-react";
+
+const CKEditorDemo = () => {
+	const cloud = useCKEditorCloud({
+	version: "43.0.0",
+	ckbox: {
+		version: "2.5.1",
+		// Optional - it's already 'lark' by default.
+		theme: "lark",
```

Comment:
not sure what we mean by optional, if it's already loads lark, then it's not needed here.

Reply:
We may, however, want to keep it to show having the parameter.